.circleci/config.yaml

```diff
@@ -0,0 +1,36 @@
+version: 2
+jobs:
+  deploy:
+    docker:
+      # Upgrade the Dockerfile and push image when the hub az cluster tiller
+      # version upgrades.
+      - image: r6by/kubectl-helm-azure
+        # Don't automatically upgrade the Hub monocular chart version for each
+        # new helm/monocular release. Instead assume we'll need testing, and
+        # set the version manually here.
+        environment:
+          CHART_VERSION: 1.4.4
+    steps:
+      - run:
+          name: Authenticate to cluster
+          command: |
+            az login --service-principal -u $SERVICE_PRINCIPAL_ID -p $SERVICE_PRINCIPAL_PASS -t $SERVICE_TENANT_ID
+            az aks get-credentials -n $CLUSTER_NAME -g $RESOURCE_GROUP
+      - checkout
+      - run:
+          name: Upgrade/install Monocular Helm chart with any Hub changed values
+          command: |
+            export MONGODB_ROOT_PASSWORD=$(kubectl get secret --namespace helm-hub helm-hub-mongodb -o jsonpath="{.data.mongodb-root-password}" | base64 -d)
+            helm init --client-only
+            helm repo add monocular https://helm.github.io/monocular
+            helm repo update
+            helm upgrade --install helm-hub monocular/monocular --version $CHART_VERSION --namespace helm-hub --set mongodb.mongodbRootPassword=$MONGODB_ROOT_PASSWORD -f config/helm-hub-values.yaml -f config/repo-values.yaml
```

Comment:
What happens if 2 PRs are merged to master in a short period and the one that is not the tip of master finishes last? The deploy will not contain the latest change merged. How can we prevent that?

Reply:
That's true. I guess we could go back to the raw git file URLs from master?

Reply:
That's a very good catch, what about instead of doing this for every push to master, we instead run this job on a schedule (e.g. every 15m). We'd need some way to figure out if something changed though, perhaps we can diff the values?

Reply:
This might do the trick? https://circleci.com/docs/2.0/skip-build/#auto-cancelling-a-redundant-build

Reply:
@jlegrone given this caveat:
>Note: Your project’s default branch will never auto-cancel builds. This feature only applies to non-workflow builds, builds triggered by pushes to GitHub, or workflow builds that use the new pipelines feature.

I suppose we could enable the pipeline feature, to allow auto-cancelling redundant builds. In fact I've used an orb successfully (that requires pipelines anyway), which would help with [this comment about only diffing the configs](https://github.com/helm/hub/pull/70#discussion_r279406832) (@prydonius that relates to your comment above - we could cron, or we could just diff the configs on pushes to master using the `CIRCLE_COMPARE_URL` from that orb?).

Reply:
Added the orb. 
- [ ] To-do: will need to enable pipelines in circle once this PR is approved and before this is merged.

Reply:
Tested on my fork 👍

Reply:
Pipelines and auto-canceling redundant builds enabled 👍